master/src/PrettyPrint.hs

```diff
@@ -155,7 +158,13 @@ instance LFresh (Reader DispInfo) where
 initDI :: DispInfo
 initDI = DI False S.empty
 
-type M a = (ReaderT DispInfo Identity) a
+newtype ReaderDispInfo a = ReaderDispInfo (ReaderT DispInfo Identity a)
+                         deriving (Functor, Applicative, Monad, MonadReader DispInfo)
+
+runReaderDispInfo :: ReaderDispInfo a -> DispInfo -> a
+runReaderDispInfo (ReaderDispInfo comp) = runReader comp
+
+type M a = ReaderDispInfo a
```

Comment:
I learned something about GHC today.  In `unbound-generics`, I don't have `-XOverlappingInstances` enabled when compiling `Unbound.Generics.LocallyNameless.LFresh`, whereas in `unbound`, the corresponding module `Unbound.LocallyNameless.Fresh` *is* compiled with overlapping instances allowed.

Consequenty when *this* module in *this* project is compiled, `instance LFresh (Reader DispInfo)` triggers an error about overlapping instances.

I like my fix (using a newtype wrapper) a little better, but I could also just turn on overlapping instances in this file and get the original code to compile unchanged.